drivers/serial/uart_sam0.c

```diff
@@ -377,6 +377,90 @@ static void uart_sam0_rx_timeout(struct k_work *work)
 
 #endif
 
+static int uart_sam0_configure(struct device *dev,
+								const struct uart_config *new_cfg)
```

Comment:
could probably use less TABs

Reply:
While it looks good on my text editor I agree that it might be a bit much for ones with more than 4 spaces per tab. Will change.